test/ipfs-factory/daemon-spawner.js

```diff
@@ -35,7 +35,7 @@ function Factory () {
 
       nodes.push(node)
 
-      callback(null, node.apiAddr)
+      callback(null, node.apiAddr.toString())
```

Comment:
cool, closing this in favour of https://github.com/ipfs/js-ipfs-api/pull/525 then :)